contracts/vaults/extensions/WrappedPlasmaVault.sol

```diff
@@ -0,0 +1,624 @@
+        if (assets == 0) revert ZeroAssetsWithdraw();
+
+        _burn(owner_, shares_);
+
+        ERC4626Upgradeable(PLASMA_VAULT).withdraw(assets, receiver_, address(this));
+
+        lastTotalAssets = totalAssets();
+
+        emit Withdraw(msg.sender, receiver_, owner_, assets, shares_);
+        return assets;
+    }
+
+    /**
+     * @notice Configures the management fee parameters for the vault
+     * @dev Updates management fee recipient and percentage through PlasmaVaultLib
+     *
+     * Configuration Flow:
+     * 1. Parameter Validation
+     *    - Validates fee account address
+     *    - Verifies fee percentage within limits
+     *    - Ensures configuration consistency
+     *
+     * 2. State Updates
+     *    - Updates fee recipient address
+     *    - Sets new fee percentage
+     *    - Persists configuration in storage
+     *
+     * 3. Fee System Impact
+     *    - Affects future fee calculations
+     *    - Influences share price computations
+     *    - Updates fee accrual parameters
+     *
+     * Security Features:
+     * - Owner-only access control
+     * - Parameter validation
+     * - State consistency checks
+     * - Safe fee calculations
+     *
+     * @param feeAccount_ Address to receive the management fees
+     * @param feeInPercentage_ Management fee percentage with 2 decimal places (10000 = 100%)
+     * @custom:security Restricted to contract owner
+     */
+    function configureManagementFee(address feeAccount_, uint256 feeInPercentage_) external onlyOwner {
+        PlasmaVaultLib.configureManagementFee(feeAccount_, feeInPercentage_);
+    }
+
+    /**
+     * @notice Configures the performance fee parameters for the vault
+     * @dev Updates performance fee recipient and percentage through PlasmaVaultLib
+     *
+     * Configuration Flow:
+     * 1. Parameter Validation
+     *    - Validates fee account address
+     *    - Verifies fee percentage within limits
+     *    - Ensures configuration consistency
+     *
+     * 2. State Updates
+     *    - Updates fee recipient address
+     *    - Sets new fee percentage
+     *    - Persists configuration in storage
+     *
+     * 3. Fee System Impact
+     *    - Affects profit-based fee calculations
+     *    - Influences performance tracking
+     *    - Updates fee distribution parameters
+     *    - Applies to future value increases
+     *
+     * Security Features:
+     * - Owner-only access control
+     * - Parameter validation
+     * - State consistency checks
+     * - Safe fee calculations
+     *
+     * @param feeAccount_ Address to receive the performance fees
+     * @param feeInPercentage_ Performance fee percentage with 2 decimal places (10000 = 100%)
+     * @custom:security Restricted to contract owner
+     */
+    function configurePerformanceFee(address feeAccount_, uint256 feeInPercentage_) external onlyOwner {
+        PlasmaVaultLib.configurePerformanceFee(feeAccount_, feeInPercentage_);
```

Comment:
I am not sure if clients will have the same constraint policy related with maxes in fees: PERFORMANCE_MAX_FEE_IN_PERCENTAGE and MANAGEMENT_MAX_FEE_IN_PERCENTAGE